.travis.yml

```diff
@@ -9,4 +9,6 @@ before_script:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
-  - npm install babel-cli -g
+  - yarn global add babel-cli
+script:
+  - yarn build
```

Comment:
I've had issues using yarn with CI before. Maybe they've been straightened out by now... but I think I'd rather go back to NPM. 

(perhaps I'm being overly cautious, but I once lost a day of work tracing down an incomprehensible CI error message that was because of a bug in Yarn. Flip Move runs pretty quick on CI anyway)